byzcoin/contracts.go

```diff
@@ -36,6 +39,40 @@ type Contract interface {
 	Invoke(ReadOnlyStateTrie, Instruction, []Coin) ([]StateChange, []Coin, error)
 	// Delete removes the current instance
 	Delete(ReadOnlyStateTrie, Instruction, []Coin) ([]StateChange, []Coin, error)
+	// Print ...
```

Comment:
Please finish this comment.

Naming is hard:

This is a function which formats an Instruction into a string. It works on all types of instructions, not just Invokes (which are the methods of contracts). It does no I/O, so it doesn't "print".

So it should be named "FormatInstruction".

(This is bike shedding, but I hope you can understand my reasoning. Naming is hard, but worth doing right.)

Reply:
I agree on the "Print" side, however this function only formats the method of an instruction, not the instruction itself. I'll go then for `FormatMethod`.